config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -42,7 +42,7 @@
 # Use new connection handling API. For most applications this won't have any
 # effect. For applications using multiple databases, this new API provides
 # support for granular connection swapping.
-Rails.application.config.active_record.legacy_connection_handling = false
+# Rails.application.config.active_record.legacy_connection_handling = false
```

Comment:
Was this change necessary?

Reply:
Dunno, I think it was automatically added like everything else? I can leave it out if desired. No idea what the impact is